static/src/javascripts/projects/commercial/modules/article-body-adverts.spec.js

```diff
@@ -108,31 +108,6 @@ describe('Article Body Adverts', () => {
                 expect(fixture.querySelector('#dfp-ad--im')).toBeTruthy();
             });
         });
-
-        it('inserts up to ten adverts when DFP returns empty merchandising components', () => {
```

Comment:
Is this test being deleted because it is not useful anymore, or because the test breaks it?

If the latter, could we `xit` or [skip it](https://facebook.github.io/jest/docs/en/api.html#testskipname-fn) so that we don't risk losing it forever?

Reply:
It is being deleted because it isn't useful anymore - we are removing the restriction on how many inline ads we are adding to articles.

Reply:
But also TIL about skip!